### PR TITLE
Add aap config handling by viper

### PR DESCRIPTION
**Pull request by jardmitrii**

### CODE REVIEW

File: app/config/config.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+)
+
+var (
+	Manager *viper.Viper = viper.New()
+)
+
```
Comment (ilie20088) [inline app/config/config.go]: I suggest to name it like ConfigManager

File: main.go [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
-	"log"
+	cfg "github.com/ilie20088/go-web-app-boilerplate/app/config"
 )
 
 func main() {
+	cfg.SetDefaults()
+	err := cfg.ReadFromFile()
+	if err != nil {
+		log.Fatal(err)
+	}
```
Comment (ilie20088) [inline main.go]: Maybe let's use init function in config file ?

File: main.go [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
 	pubRouter := app.PublicRouter()
 	http.Handle("/", pubRouter)
 
-	log.Fatal(http.ListenAndServe(":8000", pubRouter))
-}
\ No newline at end of file
+	log.Fatal(http.ListenAndServe(cfg.GetAddr(), pubRouter))
+}
```
Comment (ilie20088) [inline main.go]: How to get e.g. `db.host` from config ?
Should be used `Manager` yes ?

File: main.go (hunk 2 above)
Comment (jardmitrii) [inline main.go]: Yes, it's good idea.

File: app/config/config.go (hunk 1 above)
Comment (jardmitrii) [inline app/config/config.go]: I thought that cfg.Manager/config.Manager is explicit enough. But ConfigManager lgtm too.

**Pull request merged**